deploy/olm-catalog/kogito-operator/0.13.0/kogito-operator.v0.13.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,957 @@
+                - name: DEBUG
+                  value: "false"
+                image: quay.io/kiegroup/kogito-cloud-operator:0.13.0
+                imagePullPolicy: Always
+                name: kogito-cloud-operator
+                resources: {}
+              serviceAccountName: kogito-operator
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          - app.kiegroup.org
+          - apps.openshift.io
+          - image.openshift.io
+          - build.openshift.io
+          - rbac.authorization.k8s.io
+          - route.openshift.io
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - ""
+          resources:
+          - namespaces
+          verbs:
+          - get
+        - apiGroups:
+          - apps
+          resources:
+          - statefulsets
+          - deployments
+          - replicasets
+          verbs:
+          - '*'
+        - apiGroups:
+          - monitoring.coreos.com
+          resources:
+          - servicemonitors
+          verbs:
+          - get
+          - create
+          - list
+          - delete
+        - apiGroups:
+          - infinispan.org
+          resources:
+          - infinispans
+          verbs:
+          - get
+          - create
+          - list
+          - delete
+          - watch
+        - apiGroups:
+          - kafka.strimzi.io
+          resources:
+          - kafkas
+          - kafkatopics
+          verbs:
+          - get
+          - create
+          - list
+          - delete
+          - watch
+        - apiGroups:
+          - keycloak.org
+          resources:
+          - keycloaks
+          verbs:
+          - get
+          - create
+          - list
+          - delete
+          - watch
+        - apiGroups:
+          - apps
+          resourceNames:
+          - kogito-operator
+          resources:
+          - deployments/finalizers
+          verbs:
+          - update
+        - apiGroups:
+          - app.kiegroup.org
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        serviceAccountName: kogito-operator
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - services
+          verbs:
+          - list
+          - get
+          - watch
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - list
+          - get
+          - watch
+          - update
+          - patch
+        serviceAccountName: kogito-service-viewer
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: true
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - cloud
+  - kogito
+  - quarkus
+  - springboot
+  - operator
+  - native
+  - automation
+  labels:
+    alm-owner-kogito: kogito-operator
+    operated-by: kogito-operator.0.11.0
```

Comment:
This needs to be updated as well ...
can have a look here: https://github.com/kiegroup/kogito-cloud-operator/pull/420/files#diff-7a9d206eb5b9303a3c9c293020c798daR33

Reply:
I've updated this in the other PR, thanks GitHub for messing with my commits.